vitrina/projects/services.py

```diff
@@ -1,46 +1,94 @@
 from vitrina.users.models import User
 from vitrina.projects.models import Project
-from vitrina.orgs.services import has_perm, is_representative, Action
-from vitrina.orgs.models import Representative, Organization
 from django.db.models import Q
-from django.contrib.contenttypes.models import ContentType
 from django.db.models import QuerySet
+from django.contrib.auth.models import AnonymousUser
 
 
-def can_update_project(user: User, project: Project) -> bool:
+def _q_visible_projects(user: User | AnonymousUser) -> Q:
```

Comment:
What is q? Let's use full names
```suggestion
def _q_visible_projects(user: User | AnonymousUser) -> Q:
```